examples/duet/dcgan/DCGAN_Syft_Data_Scientist.ipynb

```diff
@@ -7,9 +7,7 @@
    "source": [
```

Comment:
<p>Why is this information is removed? Was not ok?</p><p> </p><p>input is Z, going into a convolution</p><p>&nbsp;&nbsp;&nbsp;&nbsp;----------------------------------------------------------------</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Layer (type)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Output Shape&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Param #</p><p>&nbsp;&nbsp;&nbsp;&nbsp;================================================================</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;ConvTranspose2d-1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[-1, 112, 4, 4]&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;179,200</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;BatchNorm2d-2&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[-1, 112, 4, 4]&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;224</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;ReLU-3&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[-1, 112, 4, 4]&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;0</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;ConvTranspose2d-4&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[-1, 56, 7, 7]&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;56,448</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;BatchNorm2d-5&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[-1, 56, 7, 7]&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;112</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;ReLU-6&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[-1, 56, 7, 7]&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;0</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;ConvTranspose2d-7&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[-1, 28, 14, 14]&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;25,088</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;BatchNorm2d-8&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[-1, 28, 14, 14]&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;56</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;ReLU-9&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[-1, 28, 14, 14]&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;0</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;ConvTranspose2d-10&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[-1, 1, 28, 28]&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;448</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Tanh-11&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[-1, 1, 28, 28]&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;0</p><p>&nbsp;&nbsp;&nbsp;&nbsp;================================================================</p><p>&nbsp;&nbsp;&nbsp;&nbsp;Total params: 261,576</p><p>&nbsp;&nbsp;&nbsp;&nbsp;Trainable params: 261,576</p><p>&nbsp;&nbsp;&nbsp;&nbsp;Non-trainable params: 0&nbsp;&nbsp;&nbsp;&nbsp;</p><p>&nbsp;&nbsp;&nbsp;&nbsp;"""</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/5488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>It was for the original version, just the user couldn't generate it without additionnal packages. </p><p>Also in the example above I've used the resize transform so it isn't anymore.</p>

Reply:
Thanks for the comment!